distribution/scripts/jmeter/run-performance-tests.sh

```diff
@@ -33,10 +33,10 @@ function initialize() {
 export -f initialize
 
 declare -A test_scenario0=(
-    [name]="passthrough_http"
-    [display_name]="Passthrough HTTP service"
-    [description]="An HTTP Service, which forwards all requests to a back-end service."
-    [bal]="passthrough.balx"
+    [name]="h1c_h1c_passthrough"
+    [display_name]="Passthrough HTTP service (h1c -> h1c)"
+    [description]="A HTTP Service, which forwards all requests to a HTTP back-end service."
```

Comment:
"An HTTP" should be correct. Search in https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol. There are other references as well: https://www.google.com/search?q=%22an+http%22

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-performance/pull/23/commits/445f1a74637b7636869110acc99d20af08a49923